src/Products/Annotation/Annotator/BaseAnnotator.cs

```diff
@@ -12,93 +13,92 @@ namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
     public abstract class BaseAnnotator
     {
         public string Message = "Annotation of type {0} for this file type is not supported";       
-        protected AnnotationDataEntity annotationData;
-        protected PageData pageData;
+        protected static AnnotationDataEntity annotationData;
+        protected PageInfo pageInfo;
 
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="annotationData"></param>
-        /// <param name="pageData"></param>
-        public BaseAnnotator(AnnotationDataEntity annotationData, PageData pageData)
+        /// <param name="pageInfo"></param>
+        protected BaseAnnotator(AnnotationDataEntity annotationDataArg, PageInfo pageInfo)
         {
-            this.annotationData = annotationData;
-            this.pageData = pageData;
+            annotationData = annotationDataArg;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this assignment of 'annotationData' or initialize it statically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)